modules/apps/map/map-common/src/main/resources/META-INF/resources/js/MapBase.js

```diff
@@ -100,12 +122,12 @@ class MapBase extends State {
 
 		this.on('positionChange', this._handlePositionChanged);
 
-		const geolocation =
+		const location =
```

Comment:
I wouldn't use `location` here since I believe that is a globally defined variable from `window.location`

Reply:
Alrighty!